src/main/java/com/gmail/chernii/oleksii/HibernateEntityManager.java

```diff
@@ -0,0 +1,31 @@
+package com.gmail.chernii.oleksii;
+
+import lombok.experimental.UtilityClass;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+/**
+ * Created by Space on 16.04.2019.
+ */
+@UtilityClass
+public class HibernateEntityManager {
+    private static final String PERSISTANCE_UNIT_NAME = "hibernate";
+    private static EntityManagerFactory entityManagerFactory;
```

Comment:
Please add  a blank line between lines 14 and 15 to separate constants and static fields